src/main/java/com/google/sps/servlets/YoutubeServlet.java

```diff
@@ -46,7 +47,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
       String numCommentsInput = request.getParameter(NUM_COMMENTS_PARAMETER);
       // If nothing is inputted, length of list is default, otherwise it's what was inputted.
       long numComments =
-          (numCommentsInput != null) ? Long.valueOf(numCommentsInput) : DEFAULT_COMMENTS;
+          (numCommentsInput != null)
+              ? Math.min(Long.valueOf(numCommentsInput), MAX_COMMENTS)
```

Comment:
correct me if i messed this up: can numCommentsInput gets larger than 500? I think molly's PR changes the value from 1000 to 500 already?

Reply:
It does. This is for if someone accesses the servlet link directly. For instance:
https://step31-say.uc.r.appspot.com/YouTubeComments?url=_c1NJQ0UP_Q&numComments=100
Will directly access the servlet's output.

This pr ensures that if someone put in like numComments=100000 they don't just tear through our quota

Reply:
+1, it's important to not rely on client-side validation since it's usually trivial to bypass